MAUI/ListView/viewappearance.md

```diff
@@ -78,7 +78,7 @@ Assign custom `DataTemplateSelector` to the [ItemTemplate](https://help.syncfusi
   </Grid>
 </ContentPage>
 {% endhighlight %}
-{% highlight c# %}
+{% highlight c# tabtitle= "MainPageCs.cs" hl_lines="11" %}
```

Comment:
MainPage.xaml.cs

rename the class name to MainPage

Reply:
resolved the changes